test/models/auction_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class AuctionTest < ActiveSupport::TestCase
+  test 'all_auctions returns an array of auctions' do
```

Comment:
In line with the model comments, these tests seem to be just testing ActiveRecord behavior which we can assume that it just works®.

Reply:
Personally, I am indifferent to the existence of these tests at the moment. Right now, there is effectively a 1-1 mapping between the data we store and the data that is returned. However, that will change as we add more features.